sdks/python/test-suites/dataflow/py3/build.gradle

```diff
@@ -20,7 +20,7 @@ apply plugin: org.apache.beam.gradle.BeamModulePlugin
 applyPythonNature()
 
 // Required to setup a Python 3 virtualenv.
-project.ext.python3 = true
+pythonVersion = '3.5'
```

Comment:
should the directory of this gradle file be renamed to py35 in the future?

Reply:
That's what I proposed in [this](https://lists.apache.org/thread.html/7c48200678c570c6e42d4933f8ea4f9324d82f7ae720b591a93ffcda@%3Cdev.beam.apache.org%3E) dev@ thread. Hopefully in future, tasks can be grouped by versions.